README.md

```diff
@@ -24,10 +25,26 @@ Monkeying around with virtual machines and pxe configs.
 
 ## Setup
 
-Create `marmoset.conf` before using marmoset! See `Configuration` for details.
+Clone the repo, for example into /opt. Than copy the service file into the systemd directory and reload systemd to recognize the file:
```

Comment:
"Clone the repository into a directory of your choice, /opt for example" , "then"